pkgs/servers/monitoring/plugins/default.nix

```diff
@@ -76,9 +60,6 @@ stdenv.mkDerivation rec {
   # --with-ping-command needs to be done here instead of in
   # configureFlags due to the spaces in the argument
   postPatch = ''
-    substituteInPlace po/Makefile.in.in \
-      --replace /bin/sh ${runtimeShell}
-
     sed -i configure.ac \
       -e 's|^DEFAULT_PATH=.*|DEFAULT_PATH=\"${binPath}\"|'
 
```

Comment:
the configureFlagsArray can be merged into configureFlags

Reply:
As the comment says, this does not work due to the spaces.

https://github.com/NixOS/nixpkgs/blob/d17488699f04a0a04ee4402ff22275c384508a9e/pkgs/servers/monitoring/plugins/default.nix#L60-L61

I have just verified this. The comment is still correct.